datasets/wegenbestand/dataset.json

```
@@ -0,0 +1,55 @@
+{
+  "id": "wegenbestand",
+  "type": "dataset",
+  "authorizationGrantor": "Deze gegevensset wordt onderhouden door bereikbaarheidsthermometer Amsterdam",
+  "theme": [
+    "verkeer"
+  ],
+  "homepage": "https://data.amsterdam.nl",
+  "owner": "Gemeente Amsterdam, ...",
```

This looks incomplete. Is it

```suggestion
  "owner": "Gemeente Amsterdam, bereikbaarheidsthermometer Amsterdam",
```